tests/readme-icons.test.js

```diff
@@ -0,0 +1,34 @@
+import { test, exec } from 'uvu';
+import * as assert from 'uvu/assert';
+import path from 'path';
+import * as fs from 'fs';
+import { fileURLToPath } from 'url';
+
+(async () => {
+  test(`Check readme light and dark icons`, () => {
+    const __dirname = path.dirname(fileURLToPath(import.meta.url));
+    const lightIconsPath = path.resolve(__dirname, '../assets/readme');
+    const darkIconsPath = path.resolve(__dirname, '../assets/readme');
+    fs.readdirSync(lightIconsPath).forEach((file) => {
```

Comment:
We are in async context, please use the async version of the function.

Also, `.forEach` adds more uneeded complexity here, perhaps a `for of` loop is enough.

Reply:
Sync is fine for what is needed, no need for an async + callback.

Reply:
> Since this is test code, there is no need for the performance advantage of using promises.
> A for..of is probably good enough, yes, but this code has no need for optimization.

So why we've changed the test framework from jest to esbuild in #6898? Sorry, optimizations matters a lot since our tests grows as every icon added to the library. Also, there is no need for callback, use a promise resolving with `await`. I'm going to change it.